frontend/pages/SoftwarePage/components/AddSoftwareForm/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { getPlatformDisplayName } from "utilities/file/fileUtils";
+
+import { IAddSoftwareFormData } from "./AddSoftwareForm";
+
+export const getFormSubmitDisabled = (
```

Comment:
Nit: It might make the usage clearer in the component to call this `validateFormData` or `shouldDisableFormSubmit`.

Reply:
i'll change it to `should` as its not meant to validate the data, just get the state for the button.
There will be other code that validates the form data and returns error messages.